README.md

```diff
@@ -33,24 +40,18 @@ Feel free to use any technology available in the React environment to achieve th
         "username": "test.user@phrasee.co",
         "password": "testpassword"
     }
-    A successful login will have a response with a 200 HTTP status code
-    An unsuccessful login will have a response with a 401 HTTP status code
+   
 3.  After a successful response has been received, use it to render a page listing hospital patients following the design attached in 'digital-hospital-global-patients.jpg'.
 4.  This page should not be available to users who are 'unauthenticated'. Those users should be presented with the login page instead.
-5.  Please use Redux to manage the state of the app and place any HTTP requests in action dispatcher.
+5.  Favor the use Redux to manage the some of the state of the app.
```

Comment:
This is a bit weird grammatically, can we change it to "Whilst not a strict requirement, it is preferable that you use Redux to manage at least some of the state of the app." please?

Reply:
Thanks, done